tests/e2e/pact/specs/eps.spec.ts

```diff
@@ -94,7 +96,9 @@ jestpact.pactWith(
 
       test("should be able to send a repeat-dispensing parent-prescription-1", async () => {
         const apiPath = "/Send";
-        const interaction: InteractionObject = {
+          const body = JSON.parse(sendRepeatDispensingPrescriptionSendRequest) as Bundle
+          body.identifier.value = uuid.v4()
```

Comment:
I've moved the publishing pact step into the release pipeline just before verification to support this